src/jit/codegenarm.cpp

```diff
@@ -1351,12 +1351,41 @@ void CodeGen::genLeaInstruction(GenTreeAddrMode* lea)
 //------------------------------------------------------------------------
 // genSetRegToCond: Generate code to materialize a condition into a register.
 //
+// Arguments:
+//   dstReg - The target register to set to 1 or 0
+//   tree - The GenTree Relop node that was used to set the Condition codes
+//
+// Return Value: none
+//
 // Preconditions:
 //    The condition codes must already have been appropriately set.
 //
 void CodeGen::genSetRegToCond(regNumber dstReg, GenTreePtr tree)
 {
-    NYI("genSetRegToCond");
+    // Emit code like that:
+    //   ...
+    //   bgt True
+    //   movs rD, #0
+    //   b Next
+    // True:
+    //   movs rD, #1
+    // Next:
+    //   ...
```

Comment:
I saw that comment and I was wondering the same thing. If it's good for LLVM I guess it should be good for the JIT too.

Some ARMASM manual [page](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0802b/Cjabicci.html) I was able to find does mention deprecation but not of the instruction itself but of a certain assembler syntax and the use of IT together with certain other instructions.